tests/setup_script.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-sudo apt update -y && sudo apt install -y curl
+apt update -y && sudo apt install -y curl
```

Comment:
It seems strange that only the `apt update` isn't needed. At least on my desktop Ubuntu it _is_ needed generally.